hardware_interface/test/interface_manager_test.cpp

```diff
@@ -84,6 +84,93 @@ TEST(InterfaceManagerTest, InterfaceRewriting)
   EXPECT_EQ(2, foo_iface_ptr->foo);
 }
 
+struct TestCombinationHandle {
```

Comment:
Should be class since it contains member funcs

Reply:
Applied in rebased commits